src/ui/Navbar.fs

```diff
@@ -4,67 +4,125 @@ open Feliz
+                            (if isMenuOpen then 0 else 100) |> style.filter.invert
+                        ]
                     ]
                 ]
             ]
 
             NavbarContent [
-                prop.className [ "sm:flex"; "gap-4" ]
+                prop.className [ "hidden"; "sm:flex"; "gap-4" ]
+                prop.style [ length.vw 5 |> style.gap ]
                 justify "center"
                 prop.children [
+
                     NavbarItem [
                         Link [
-                            prop.text "Products"
-                            color "foreground"
-                            prop.onClick (fun _ -> Router.navigatePath "/products")
-                            prop.style [ style.cursor "pointer" ]
+                            prop.text "Shop"
+                            prop.onClick (fun _ -> Router.navigatePath "/")
+                            prop.style [ style.fontSize 13; style.cursor "pointer"; style.color "white" ]
                         ]
                     ]
 
                     NavbarItem [
-                        prop.children [
-                            Link [
-                                prop.text "About Us"
-                                color "foreground"
-                                prop.onClick (fun _ -> Router.navigatePath "/about")
-                                prop.style [ style.cursor "pointer" ]
-                            ]
+                        Link [
+                            prop.text "About Us"
+                            prop.onClick (fun _ -> Router.navigatePath "/about")
+                            prop.style [ style.fontSize 13; style.cursor "pointer"; style.color "white" ]
                         ]
                     ]
 
                     NavbarItem [
                         Link [
                             prop.text "Contact"
-                            color "foreground"
                             prop.onClick (fun _ -> Router.navigatePath "/contact")
-                            prop.style [ style.cursor "pointer" ]
+                            prop.style [ style.fontSize 13; style.cursor "pointer"; style.color "white" ]
                         ]
                     ]
                 ]
             ]
 
             NavbarContent [
                 justify "end"
+                prop.style [ length.px 40 |> style.gap ]
                 prop.children [
                     NavbarItem [
-                        Button [ color "primary"; prop.custom ("variant", "flat"); prop.text "Cart" ]
+                        img [
+                            prop.src <| import "default" "./assets/Cart.svg"
+                            prop.onClick (fun _ -> Router.navigatePath "/")
+                            prop.style [
+                                length.px 21 |> style.height
+                                style.cursor "pointer"
+                                (if isMenuOpen then 0 else 100) |> style.filter.invert
```

Comment:
Again, a transition here to stop the disappearance would probably be a good thing.